stl/inc/generator

```diff
@@ -231,20 +231,23 @@ public:
         requires same_as<_Gen_yield_t<_Gen_reference_t<_Rty, _Vty>>, _Yielded>
     _NODISCARD auto yield_value(_RANGES elements_of<generator<_Rty, _Vty, _Alloc>&&, _Unused> _Elem) noexcept {
         using _Nested_awaitable = _Nested_awaitable_provider<_Rty, _Vty, _Alloc>::_Awaitable;
-        return _Nested_awaitable{std::move(_Elem.range)};
+        return _Nested_awaitable{_STD move(_Elem.range)};
     }
 
     template <_RANGES input_range _Rng, class _Alloc>
         requires convertible_to<_RANGES range_reference_t<_Rng>, _Yielded>
     _NODISCARD auto yield_value(_RANGES elements_of<_Rng, _Alloc> _Elem) {
         using _Vty              = _RANGES range_value_t<_Rng>;
         using _Nested_awaitable = _Nested_awaitable_provider<_Yielded, _Vty, _Alloc>::_Awaitable;
-        return _Nested_awaitable{[](allocator_arg_t, _Alloc, _RANGES iterator_t<_Rng> _It,
-                                     const _RANGES sentinel_t<_Rng> _Se) -> generator<_Yielded, _Vty, _Alloc> {
+
+        auto _Lambda = [](allocator_arg_t, _Alloc, _RANGES iterator_t<_Rng> _It,

```

Comment:
Maybe make this a function? This lambda does not capture.
(Remember that lambdas contribute to code bloat when not inline, as they have unique real type per TU)

Reply:
Yep, I have to do this for Clang (which doesn't permit lambdas to be converted to `__stdcall` function pointers under `-fno-ms-compatibility`).